wordpress-mobile/WordPress-iOS
# [Gutenberg] File block IV - Error handling, retry, and cancel upload

This PR updates the gutenberg ref to test https://github.com/WordPress/gutenberg/pull/27146

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/27146
`gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2826

![Error](https://user-images.githubusercontent.com/9772967/99811631-b5eedc80-2b45-11eb-8aaa-51395acfcc84.png)

**Note:** This PR is not intended to be merged

**Note 2:** I have created a temporal JS Bundle with File block active, so this PR can be tested using the **CI installable build**.

### To test:

**Test cancel action**:
- Add a File block to an empty new post.
- Add a (big-ish) file to the block.
- While the file is uploading, tap on the block's background to see the cancel action sheet.
  - If you tap on the file name, it will start editing the file name fileld. This is intended (cc @iamthomasbishop )
- Tap the `Stop upload` action.
  - The File block should go back to the empty state.

**Test error and retry**:
- Add a File block to an empty new post.
- While it's uploading, turn un Airplane mode.
  - See the block switch to its error state.
- While in the error state, tap on the File block background to see the Retry action sheet.
- After reconnecting the device, press the Retry action.
  - Check that the file starts uploading again until it successfully finishes uploading.

PR submission checklist:
